templates/community-viz/dist/index.html

```diff
@@ -1,10 +1,19 @@
 <!doctype html>
 <html>
   <head>
-    <title>Getting Started</title>
+    <title>Community Viz</title>
+    <style>
+      body {
+        font-family: Roboto;
+      }
+      code {
+        background-color: #fdf6e3	;
+        color: #586e75;
+      }
+    </style>
   </head>
   <body>
-    <iframe src = "vizframe.html" componentid="abc" width="50%" height = "500px">
```

Comment:
So we're not making this inside an iframe anymore? Does `dscc.getWidth()` and `dscc.getHeight()` still work without the iframe?

Reply:
Adding the iframe back in